### ISSUE TITLE
Examples don't display properly. Some parts are missing (Dots, brackets and more)

### ISSUE THREAD

**Issue by fennekbe** [nodeschool/discussions#1567]
Either way, I'm having a problem where the nodeschool assignments don't display the examples properly. For example console.log('test') will display as console log 'test 'which makes it rather hard to follow the examples. Any suggestions?

Thank you

**Comment by mauaunot** [comment]
hi, I have the same problem, some solution?, i have node -v 0.12.4
thanks

**Comment by fennekbe** [comment]
0.12.4? How does that work? I have v5.1.0

**Comment by mauaunot** [comment]
v0.12.4 is an earlier version to version 4 (io.js + node.js). the important thing is that we have the same problem with both versions

regards

**Comment by fennekbe** [comment]
I'm sorry, I'm new to node. I had seen version 0.12.4 mentioned before but I was clueless as to what it meant.

**Comment by mauaunot** [comment]
Don´t worry, I´m also new to Node. I hope someone help us with the problem
regards

**Comment by alagiboy** [comment]
Hi, I had the same issue but mine was an issue with the font color, some of the text/characters are using same color as terminal background.
 
You can verify if the case is the same for you by highlighting the example, if thats the case then easy fix would be to change your terminal background.

**Comment by XLCM** [comment]
Similar issue: I was doing the Javascripting track and had to copy the activity text out into Notepad.  Changing the terminal background color didn't reveal everything.

**Comment by fennekbe** [comment]
actually 'color f' works for me. Thanks

**Comment by mauaunot** [comment]
Thanks, copy and paste from the console to notepad has helped me to keep going
Regards :)
